src/commands/preload/index.ts

```diff
@@ -192,10 +192,12 @@ Can be repeated to add multiple certificates.\
 				event.name,
 			));
 			if (event.action === 'start') {
-				return spinner.start();
+				spinner.start();
+				return;
 			} else {
```

Comment:
I know this was here before but we don't even need the `else` since we `return`. We can just call `spinner.stop()` right after. We also don't even need that last return after the `spinner.stop` I think 🤔

Reply:
Sure. This was auto corrected by the linter, but fixed anyway